src/DIRAC/WorkloadManagementSystem/Client/PilotStatus.py

```diff
@@ -0,0 +1,48 @@
+"""
+This module contains constants and lists for the possible pilot states.
+"""
+__RCSID__ = "$Id$"
+
+# The pilot has been generated and is transferred to a remote site:
```

Comment:
```suggestion
#: The pilot has been generated and is transferred to a remote site:
```
Also for all the other constants, 
then the strings and constants show up in the documentation, instead of empty

https://dirac--5314.org.readthedocs.build/en/5314/CodeDocumentation/WorkloadManagementSystem/Client/PilotStatus.html